### PR TITLE
Add various fixes

### PR DESCRIPTION
**Pull request by francas**
Documentation content update for [Module 7-cosmjs, Section 4-with-keplr]

This PR fixes multiple things:
- typos
- moves `gasPriceStep` property under `feeCurrencies`
- removes protocol from rpcUrl - with https it threw CORS error, with http it just returned 403 forbidden
- removes `OfflineSigner` type as it was invalid

### Change scope

The changes in this Pull Request include (please tick all that apply):

* [x] Small language/grammar fixes
* [ ] Small content fixes 
* [ ] Addition of new content
* [x] Sample code/command updates
* [ ] Platform fixes
* [ ] Other

### CODE REVIEW

File: tutorials/7-cosmjs/4-with-keplr.md [hunk 1]
```diff
@@ -306,11 +306,6 @@ getTestnetChainInfo = (): ChainInfo => ({
         coinGeckoId: "cosmos",
     },
     coinType: 118,
-    gasPriceStep: {
-        low: 1,
-        average: 1,
-        high: 1,
-    },
```
Comment (xavierlepretre) [inline tutorials/7-cosmjs/4-with-keplr.md:313]: Did this not work for you? 
I think this is a new Keplr thing. What's your Keplr version?

File: tutorials/7-cosmjs/4-with-keplr.md [hunk 2]
```diff
@@ -153,7 +153,7 @@ import { FaucetSender } from '../components/FaucetSender'
 const Home: NextPage = () => {
   return <FaucetSender
     faucetAddress="cosmos15aptdqmm7ddgtcrjvc5hs988rlrkze40l4q0he"
-    rpcUrl="https://rpc.sentry-01.theta-testnet.polypore.xyz" />
+    rpcUrl="rpc.sentry-01.theta-testnet.polypore.xyz" />
```
Comment (xavierlepretre) [inline tutorials/7-cosmjs/4-with-keplr.md:156]: Did this not work for you?

File: tutorials/7-cosmjs/4-with-keplr.md [hunk 3]
```diff
@@ -360,7 +355,7 @@ onSendClicked = async(e: MouseEvent<HTMLButtonElement>) => {
     // Suggest the testnet chain to Keplr
     await keplr.experimentalSuggestChain(this.getTestnetChainInfo())
     // Create the signing client
-    const offlineSigner: OfflineSigner =
+    const offlineSigner =
```
Comment (xavierlepretre) [inline tutorials/7-cosmjs/4-with-keplr.md:358]: I prefer to leave this in to give additional information. Compiler errors can be fixed with the proper `import`.

File: tutorials/7-cosmjs/4-with-keplr.md (hunk 1 above)
Comment (francas) [reply tutorials/7-cosmjs/4-with-keplr.md:313]: Yes, this property no longer exists in `ChainInfo` type. I have 
```
    "@keplr-wallet/types": "^0.11.23"
```

File: tutorials/7-cosmjs/4-with-keplr.md (hunk 2 above)
Comment (francas) [reply tutorials/7-cosmjs/4-with-keplr.md:156]: Yes, with https I got CORS error, with http - 403 Forbidden, it worked only by removing it.

File: tutorials/7-cosmjs/4-with-keplr.md (hunk 3 above)
Comment (francas) [reply tutorials/7-cosmjs/4-with-keplr.md:358]: That's the thing, `OfflineSigner` is no longer a valid type. Maybe it should then be updated to `OfflineAminoSigner & OfflineDirectSigner` as that's the type returned by `window.getOfflineSigner`

Comment (xavierlepretre) [reply tutorials/7-cosmjs/4-with-keplr.md:358]: Ah, then the hands-on exercise will be updated in time when we review the whole CosmJS exercise. And for this unlinked code, we might as well change like your suggestion.

File: tutorials/7-cosmjs/4-with-keplr.md (hunk 1 above)
Comment (xavierlepretre) [reply tutorials/7-cosmjs/4-with-keplr.md:313]: Ah I see here https://docs.keplr.app/api/suggest-chain.html that the `gasPriceStep` has been moved into `feeCurrencies[x]`. Can you put it there?

Comment (francas) [reply tutorials/7-cosmjs/4-with-keplr.md:313]: Sure, done

**Review by coldice — APPROVED** [review]

**Pull request merged**